cli/azd/pkg/exec/runargs.go

```diff
@@ -32,6 +32,13 @@ type RunArgs struct {
 
 	// When set will call the command with the specified StdOut
 	StdOut io.Writer
+
+	// When set, any calls within the command to `az` will be sent to `azd`
+	AzEmulator bool
```

Comment:
Do we see a different use case than terraform? Personal preference here is to leave it as feature-specific for now, we can make this a general feature if the use case does arise.

Reply:
Makes sense

Reply:
removed it